src/navigator/types.tsx

```diff
@@ -277,8 +276,7 @@ export type StackParamList = {
   [Screens.TabHome]: { isTabNavigator?: boolean } | undefined
   [Screens.TabWallet]: { activeAssetTab?: AssetTabType; isWalletTab?: boolean } | undefined
   [Screens.TabNavigator]: {
-    initialScreen?: Screens
-    fromModal?: boolean
+    initialScreen?: Screens.TabHome | Screens.TabWallet | Screens.TabDiscover
```

Comment:
only doing this for tab navigator as we plan to remove drawer soon after we roll this out